app/_data/speakers_info.yml

```diff
@@ -0,0 +1,129 @@
+- firstName: Avdi
+  lastName: Grimm
+  shortDesc: Founder of RubyTapas
+  twitter: avdi
+  github: avdi
+  bio: Avdi Grimm has been hacking Ruby code for over 10 years, and is still loving it. He is chief aeronaut at ShipRise, head chef at RubyTapas.com, and a Ruby Rogue. He lives in Eastern Tennessee with his wife and a steadily incrementing number of children, and in his copious spare time blogs and podcasts at Virtuous Code and Wide Teams.
+  talk_title: TBA
+  talk_description: TBA
+  picture_name: avdi.png
+  website:
```

Comment:
Yea, I did what you said and put it in another file but I was already counting on just keeping a single one. I'll have that updated in a bit.